### PR TITLE
changelog: update for upcoming 19.10.1 release

### LINKED ISSUE #1461: Prep for 19.10 release
@clobrano @Feichtmeier just wanted to file this to chat about making the first `19.10.1` release, think you've been talking to @3v1n0 about this already?

Could one of you write the Debian changelog maybe please? Anything else that needs doing?

### PR DESCRIPTION
**Pull request by clobrano**
Closes #1461

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,71 @@
+yaru-theme (19.10.1) eoan; urgency=medium
```
Comment (3v1n0) [inline debian/changelog]: Replace `eoan` with UNRELEASED, we can then do the changelog finalization + tagging once the package went in eoan for real.

**Comment by 3v1n0** [comment]
A part from the comment, I think this is  good, thanks.

Once merged we'll prepare the package and push a release commit.

**Comment by 3v1n0** [comment]
LGTM, just rebase on master (in case adding missing changelog lines), and squash your changes to a single commit then I think we're good to go.

**Comment by clobrano** [comment]
master hasn't changed since the changelog branch, so I think it's fine now. Merging it

**Pull request merged**